indexing/metadata.md

```diff
@@ -18,44 +18,199 @@ Including the following tags and metadata in each of your pages will improve the
+<meta name="dc.date.created" content="YYYY-MM-DD" />`  
+<meta name="dcterms.created" content="YYYY-MM-DD" />
+```
+- **Detail:** The date fields are listed in order of preference. We only accept one `created` date, and use this list to determine the fallback options. Exact time is optional; [read more here](https://en.wikipedia.org/wiki/ISO_8601).
+- **Used In:** Page freshness scoring, faceted search (upcoming feature)
+
+
+### Changed Date
+
+We look for the following fields:
+```
+<meta property="article:modified_time" content="YYYY-MM-DD" />
+<lastmod>
+```
+
+- **Detail:** Exact time is optional; [read more here](https://en.wikipedia.org/wiki/ISO_8601). The `<lastmod>` field is included in XML sitemaps to signal to search engines when a page was last modified. Search.gov collects this metadata in case there is no `article:modified_time` data included in the page itself.
+- **Used In:** Page freshness scoring, faceted search (upcoming feature)
+
+
+### Crawling Metadata
+
+We look for the following field:
+```
+<meta name="robots" content="..., ..." />
+```
+- **Detail:** Use the [meta robots tag]({{ site.baseurl }}/indexing/how-search-engines-index-content-better-discoverability.html#robots) to block the search engine from indexing a particular page.
+- **Used In:** Used during indexing, does not affect relevance ranking.
+
+
+
+
+## Upcoming Metadata Support
+The following fields will power new features coming in the next few months. These will not impact your search until we release those features. We recommend adding these fields to your documents *now* to use these new features sooner. 
+
+### Thumbnail Image
+
+We look for the following field:
+```
+<meta property="og:image" content="https://domain.com/my-image-url.jpg" />
+```
+- **Detail:** An image associated to the content. Ideally, this image serves well as a thumbnail on both our search results page and on social media websites like Facebook and Twitter.
+- **Used In:** Results page display (upcoming feature)
+
+
+### Audience
+
+We look for the following field:
+```
+<meta name="dcterms.audience" content="example">
+```
+- **Detail:** The intended audience of the page. 
+- **Used In:** Query matching, faceted search (upcoming feature)
+
+
+### Content Type
+
+We look for the following fields:
+```
+<meta name="dc.type" content="example">
+<meta name="dcterms.type" content="example">
+```
+- **Detail:** The content type of the page. We combine these fields into one "Content Type" field for searching and filtering.
+- **Used In:** Faceted search (upcoming feature)
+
+
+### Search.gov Custom Fields
+
+We look for the following fields:
+```
+<meta name="searchgov_custom1" content="..., ..., ...">
+<meta name="searchgov_custom2" content="..., ..., ...">
+<meta name="searchgov_custom3" content="..., ..., ...">
+```
+- **Detail:** These fields can support any text content that you want to surface in your faceted search options. The fields accept single values or list-type content. 
```

Comment:
```suggestion
- **Detail:** These fields can support any text content that you want to surface in your faceted search options. The fields accept single values or comma-separated list-type content. If a comma-separated list is supplied, each item in that list will be treated as a unique term when searching/filtering by this content.
```

(Above could probably be worded better, but the same logic also applies to `tags` content however we want to phrase it.)